common/sign/v1/iss.c.inc

```diff
@@ -74,9 +74,9 @@ int _ISS_PREFIX(key_digest)(trit_t *key, trit_t *digest,
 
 int _ISS_PREFIX(address)(trit_t const *const digest, trit_t *const address,
                          size_t const digest_length, HASH_STATE *const state) {
+  _HASH_PREFIX(reset)(state);
```

Comment:
Please undo; we expect the state to have been reset before calling this method.
Also, all ISS methods in entangled clean up after themselves, hence the reset at the end of the method.